packages/gatsby-remark-images/src/index.js

```diff
@@ -190,6 +190,15 @@ module.exports = (
       )
     }
 
+    const imageStyle = `
+      width: 100%;
+      height: 100%;
+      margin: 0;
+      vertical-align: middle;
+      position: absolute;
+      top: 0;
+      left: 0;`.replace(/\s*(\S+:)\s*/g, `$1`)
```

Comment:
Isn't this just `.replace(/\s*/g, '')` ? Or are we defensive about future updates where certain rules may be whitespace sensitive.

Reply:
(Also, if you're stripping whitespace anyways, it's probably nice to have the closing backtick on the next line, aligned with the `const`.)

Reply:
Thanks for the review!

Not sure about the regexp. I copied it in from the [older version of the file](https://github.com/gatsbyjs/gatsby/blob/9fb78f8634d42e0f5547e320aafed9ae5b489223/packages/gatsby-remark-images/src/index.js#L177). I think being defensive here is a good move, though.

Since the regexp only strips before the CSS property name and between the colon and the value, it doesn’t strip the newline after the last line, so leaving backtick as-is.